internal/schema-registry/command_dek.go

```diff
@@ -0,0 +1,52 @@
+package schemaregistry
+
+import (
+	"github.com/spf13/cobra"
+
+	srsdk "github.com/confluentinc/schema-registry-sdk-go"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *command) newDekCommand(cfg *config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "dek",
+		Short:       "Manage Schema Registry DEK.",
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLoginOrOnPremLogin},
+	}
+
+	cmd.AddCommand(c.newDekCreateCommand(cfg))
+	cmd.AddCommand(c.newDekDeleteCommand(cfg))
+	cmd.AddCommand(c.newDekDescribeCommand(cfg))
```

Comment:
a name and a subject will specify just one dek. If just specify a name, you can see the subjects by `sr dek subject list`

Reply:
Shouldn't we be able to list all of the DEKs if we don't specify a name and subject though?

Reply:
(Maybe we don't have an API for this, but it seems like it would be helpful if we did!)

Reply:
sounds good
do we have other commands that do something similar?

Reply:
this will not be efficient to do imo. to get a DEK it needs a kek name, a subject and a version. sounds like:
1. list all KEK names.
2. list all sr subjects and enumerate all of them with each KEK name, as we don't know which subjects are being used.
3. ....